### PR TITLE
Data release 2021_11_19_23_36_10

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: test/integration/golden_response/get_place_page_data/asm.sample.json [hunk 1]
```diff
@@ -6927,12 +6935,15 @@
...
-            "2019": 5
+            "2019": 4,
+            "2020": 11,
+            "2021": 8
           },
           "metadata": {
             "importName": "CycloneNOAA",
-            "provenanceUrl": "https://www.ncdc.noaa.gov/",
-            "measurementMethod": "DataCommonsAggregate"
+            "provenanceUrl": "https://www.ncdc.noaa.gov/ibtracs/index.php?name=ib-v4-access",
```
Comment (shifucun) [inline test/integration/golden_response/get_place_page_data/asm.sample.json:6944]: This is expected from the cyclone data refresh

File: test/integration/golden_response/get_place_page_data/county.sample.json [hunk 2]
```diff
@@ -15034,6 +15034,27 @@
             "observationPeriod": "P1D"
           }
         },
+        "Annual_Emissions_CarbonDioxide_NonBiogenic": {
```
Comment (shifucun) [inline test/integration/golden_response/get_place_page_data/county.sample.json:15037]: These are from the landing page config update

**Review by pradh — APPROVED** [review]
This looks good.

To confirm, there are golden files related to temperature right? And there isn't any diffs with them?

I originally thought the default model for US might change, but then the mixer ranking specifies the CCSM4 model - https://github.com/datacommonsorg/mixer/blob/13f1c4a3b21e14e02e1e75cf0788f537fa6b88d6/internal/server/ranking.go#L62-L65.  But also there are more import-names, and so we need to revisit that..

**Comment by shifucun** [comment]
> This looks good.
> 
> To confirm, there are golden files related to temperature right? And there isn't any diffs with them?
> 
> I originally thought the default model for US might change, but then the mixer ranking specifies the CCSM4 model -
> 
> https://github.com/datacommonsorg/mixer/blob/13f1c4a3b21e14e02e1e75cf0788f537fa6b88d6/internal/server/ranking.go#L62-L65
> 
> . But also there are more import-names, and so we need to revisit that..

Update ranking sounds good. Also the map tool now show dropdown selection of all the sources.

**Pull request merged**